Tests/selenium/CheckCreateUser.php

```diff
@@ -0,0 +1,147 @@
+<?php
+  /**
+   * Pre-requisites: phpunit, phpunit-selenium, selenium-standalone-server, chrome driver, php-curl extension
+   * 
+   * 
+   * @Matrix Israel Ltd.
+   */
+require_once __DIR__.'/../../vendor/autoload.php';
+
+class CheckCreateUserTest extends PHPUnit_Extensions_Selenium2TestCase
+{
+    const BROWSER = "chrome";
```

Comment:
It was just an example. Selenium is compatible with most of browser via appropriate drivers.

http://www.seleniumhq.org/about/platforms.jsp